samples/bluetooth/iso_broadcast/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_BT=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_ISO=y
+CONFIG_BT_ISO_BROADCAST=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_DEVICE_NAME="Test ISO Broadcaster"
+
+# Temporary, enable the following to meet BT_ISO dependencies
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PER_ADV_SYNC=y
```

Comment:
`CONFIG_BT_ISO_BROADCAST` `select`s `CONFIG_BT_PER_ADV_SYNC` so this shouldn't be necessary.

Reply:
Yes, this can be removed.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/34219#discussion_r611581592